backend/pkg/helm/release.go

```diff
@@ -294,8 +300,13 @@ func (h *Handler) UninstallRelease(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = h.setStatus("uninstall", req.Name, processing, nil)
+	if err != nil {
+		zlog.Error().Err(err).Str("action", "uninstall").Msg("failed to set status")
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
+
```

Comment:
We should avoid redundant `failed` keyword in error message. Also how about?

```suggestion
if err = h.setStatus("uninstall", req.Name, processing, nil); err != nil {
		zlog.Error().Err(err).Str("action", "uninstall").Msg("setting status")
		http.Error(w, err.Error(), http.StatusInternalServerError)
	}
```